packages/proxy/lib/http/util/regex-rewriter.ts

```diff
@@ -1,23 +1,48 @@
+import type { SecurityOpts } from './rewriter'
+
 const pumpify = require('pumpify')
 const { replaceStream } = require('./replace_stream')
 const utf8Stream = require('utf8-stream')
+const integrityStrippedAttributeTag = 'cypress-stripped-integrity'
```

Comment:
we are referencing this a lot. Any ideas where to move this to a single source of truth?

Reply:
I took a stab at this in [f0fbf33](https://github.com/cypress-io/cypress/pull/22568/commits/f0fbf33b8950347938d7980bda534431813fc2f6). I had to opt for a json file since the directory cleans any JS files out of the lib directory when the build is run. Not awesome, but it does work and gives a single source of truth on the tag name